esphome/components/spi/spi.h

```diff
@@ -170,7 +184,7 @@ class SPIComponent : public Component {
   GPIOPin *active_cs_{nullptr};
   SPIClass *hw_spi_{nullptr};
   uint32_t wait_cycle_;
-};
+};  // namespace spi
```

Comment:
Rogue comment added?
```suggestion
};
```

Reply:
Yep